VideoAPI/Video.API/Validations/AddHeartbeatRequestValidation.cs

```diff
@@ -0,0 +1,9 @@
+﻿﻿using FluentValidation;
+using VideoApi.Contract.Requests;
+
+namespace Video.API.Validations
+{
+    public class AddHeartbeatRequestValidation : AbstractValidator<AddHeartbeatRequest>
+    {
```

Comment:
Do we really not need to validate input? Or even check that it's there?

Reply:
I have to have an empty one as the 2 required guids come from the route, all others are are defaulted and allow nulls. But the framework does not allow me NOT TO HAVE this